features/http/dashboard.feature

```diff
@@ -0,0 +1,25 @@
+@nova @dashboard @auth @views
+Feature: Test if dashboard works
+
+  Background:
+    Given there are users created:
+      | id                                   | email             | is_admin |
+      | 00000000-0000-0000-0000-000000000000 | admin@example.com | 1        |
+      | 00000000-0000-0000-0000-000000000001 | user@example.com  | 0        |
+
+  Scenario: User is requesting nova dashboard when he is not logged in
+    Given a user is requesting "/dashboard"
+    When a request is sent
+    Then a response status code should be "302"
```

Comment:
Why `302`?

Reply:
Isn't it default Nova behaviour? To throw `302 Redirect` if user is not logged in?

The same is in:
https://github.com/brewmap/brewmap/blob/97901c53a6a9d49e12526f554ad0e1b3f6ca301a/features/http/homepage.feature#L9-L12

Maybe we should remove this scenario from `homepage.feature` then?

Reply:
Ah, okay!